docs/bot/configuration.md

```diff
@@ -43,3 +43,43 @@ These are the keys you can specify:
   "contributors": []
 }
 ```
+
+#### Moving The All Contributors Table Into Different File
+
+As you know, the default file for the contributors table is `README.md`. But, if your project have tons of contributors, it doesn't make sense to includes the All Contributors Table in the `README.md` file. Instead, you can use a different file to do so.
+
+Here is the steps that may help you in achieving that. For the sake of this tutorial, the different file that we're going to be using is going to be named as `CONTRIBUTORS.md`.
+
+1. Install the bot. Read the [Installation]() page.
```

Comment:
```suggestion
1. Install the bot. See the [Installation]() page for instructions.
```

Reply:
Also, the links appear to be missing the URL part inside the `()`.

Reply:
Oh for the links. I actually don't know how Docusaurus local links work. But, after some digging from the website source file, I think I know it now.